lntest/itest/assertions.go

```diff
@@ -1362,7 +1361,7 @@ func assertDLPExecuted(net *lntest.NetworkHarness, t *harnessTest,
 			return fmt.Errorf("unable to get carol's balance: %v", err)
 		}
 
-		carolBalance := carolBalResp.ConfirmedBalance
+		carolBalance := carolBalResp.TotalBalance
```

Comment:
why the change from confirmed to total?

Reply:
This is for Neutrino only. There the sweep TX for the (non-existent) remote anchor doesn't get rejected by the backend so the wallet still sees it as being an unconfirmed balance. Maybe I should make it more explicit and add a comment here.